craft_parts/executor/part_handler.py

```diff
@@ -57,7 +57,8 @@ def __call__(
         *,
         stdout: Stream,
         stderr: Stream,
-    ) -> StepState: ...
+    ) -> StepState:
```

Comment:
Linter failure is because this is undoing the changes for black 2024 added in https://github.com/canonical/craft-parts/pull/629